parsl/tests/test_error_handling/test_resource_spec.py

```diff
@@ -26,22 +27,24 @@ def test_resource(n=2):
     fut = double(n, parsl_resource_specification=spec)
     try:
         fut.result()
-    except InvalidResourceSpecification:
-        assert isinstance(executor, HighThroughputExecutor)
+    except InvalidResourceSpecificationError:
+        assert (
+            isinstance(executor, HighThroughputExecutor) or
+            isinstance(executor, WorkQueueExecutor) or
+            isinstance(executor, ThreadPoolExecutor))
```

Comment:
Now that work queue executor exceptions are covered in this `except` block, the `except UnsupportedFeatureError` block isn't relevant any more, and you can delete it. The only exception to expect here  now is the invalid resource exception. (that is the point of this PR!)

Reply:
I thought it would be better to remove it if we gonna remove the `UnsupportedFeatureError`. I agree that is irrelevant from here but i believe irrespective of this Pr, that should always be true.